apps/nova/src/components/dashboard/PastLessons.tsx

```diff
@@ -11,27 +11,57 @@ import { InView } from "react-intersection-observer";
 import dayjs from "dayjs";
 import { Loading } from "@litespace/luna/Loading";
 
-function asLessons(list: ILesson.FindUserLessonsApiResponse["list"] | null) {
+function asLessons(
+  list: ILesson.FindUserLessonsApiResponse["list"] | null,
+  userRole?: IUser.Role
+) {
   if (!list) return [];
 
   return list.map((lesson) => {
     const teacher = lesson.members.find(
```

Comment:
We don't use the term `teacher`. Use `tutor` instead. 

```suggestion
    const tutor = lesson.members.find(
```